db/seeds.rb

```diff
@@ -31,6 +31,14 @@
   r.time_zone = 'America/New_York'
 end.save!
 
+Location.find_or_initialize_by(
+  city_name: 'SD'
```

Comment:
We have a change coming where we will access these locations from the URL. What do you think of `SanDiego` for the name to match Boulder and Providence?

Reply:
I wrote 'SD' to match the abbreviation format similar to 'NYC' and 'DC'. If it would be more clear and consistent with 'Boulder' and 'Providence' to change 'SD' to `SanDiego`, then I agree with the change.